### ISSUE TITLE
Tank pressures not imported correctly from UDDF

### ISSUE THREAD

**Issue by SeppoTakalo** [Subsurface-divelog/subsurface#4360]
### Describe the issue:

- [x] Bug
- [ ] Change request
- [ ] New feature request
- [ ] Discussion request

### Issue long description:
When exporting a dive from Shearware Cloud using UDDF format and importing that dive to Subsurface I noticed that only the first tank pressure transmitter data is used. Second tank pressure is only shown as a straight line in the graph.

![image](https://github.com/user-attachments/assets/6a4fea41-20bc-49e7-9dfe-8eb5f754cf6b)

From the log I can see that the data is still there, so it is exported properly.
<img width="481" alt="image" src="https://github.com/user-attachments/assets/4544ddaf-a291-49e0-a41c-eebcf2f0339e">

### Operating system:

Mac OS 15.1
Apple Macbook Air M3

### Subsurface version:

6.0.5283-CICD

### Steps to reproduce:

* Export UDDF from Shearwater Cloud
* Import into Subsurface

### Expected behavior:

Expect to see all tank pressures in the graph.

**Comment by mikeller** [comment]
Yes, this is a known shortcoming of the current UDDF import in Subsurface:

https://github.com/subsurface/subsurface/blob/0e7d3a6466be86f281b49564137f3a7001f7a093/xslt/uddf.xslt#L550-L554

I.e. only one tankpressure is looked at, and the `ref` attribute is completely ignored.
